catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/feeds/FeedResource.java

```diff
@@ -187,6 +203,22 @@ public Response addPost(@Context UriInfo uriInfo, @PathParam("id") String id, @V
     return Response.created(thread.getHref()).entity(thread).build();
   }
 
+  @GET
+  @Path("/{id}/posts")
+  @Operation(
+      summary = "Get all the posts of a thread",
+      tags = "feeds",
+      description = "Get all the posts of an existing thread.",
+      responses = {
+        @ApiResponse(
+            responseCode = "200",
+            description = "The posts of the given thread.",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = PostList.class))),
+      })
+  public PostList getPosts(@Context UriInfo uriInfo, @PathParam("id") String id) throws IOException {
+    return new PostList(dao.listPosts(id));
```

Comment:
I presume we do not have href for posts.

Reply:
That is correct